app/models/tasklist/SectionTitleKeys.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.tasklist
+
+import models.{Enumerable, WithName}
+
+sealed trait SectionTitleKeys extends Enumerable.Implicits
+
+object SectionTitleKeys extends SectionTitleKeys {
+
+  case object AboutYouTitleKey extends WithName("aboutYou") with SectionTitleKeys
```

Comment:
Add "TitleKey" at the end, to differentiate with similar names for section title